### DISCUSSION TITLE
How to map decoded bits received after a end to end system on constellation diagram or on a simple IQ plane ?

### DISCUSSION THREAD

**Discussion by omsahoowork** [NVlabs/sionna#399] [Q&A]
I want to see how plot changes on varying snr values since ber decreases on increasing snr value so the IQ plot should change accordingly I think.
Even how to achieve this using the baseline (perfect csi or ls estimate model) also would help...

**Comment by SebastianCa** [comment]
Hi @OmSahoo-Work ,
this discussion seems related to #397. 

You can use the example from the [MIMO tutorial notebook](https://nvlabs.github.io/sionna/examples/Simple_MIMO_Simulation.html#Simple-uncoded-transmission) to plot the received constellations. 

If you want to visualize the decoded bits, you need map the bits (and possibly re-encode) to symbols before plotting the constellation. As explained in #397, it is not clear to me what you expect to see by this experiment (bits at the decoder output are hard-decided).
